### ISSUE TITLE
Sensible defaults for permissions (role names)

### ISSUE THREAD

**Issue by cedricwaxwing** [gnosis/tabula#188]
EX:
Contributor
Admin
Custom, etc

**Comment by cedricwaxwing** [comment]
Thinking a bit more on this one, and I'm wondering if we should change "Permissions" to "Members" or something, since it's showing each member. Then, when setting permissions for the member, there are 3 or 4 options:
- Contributor
- Editor?
- Admin
- Create Custom 

If custom is selected, the entire array of checkboxes is visible, along with a "role name" field so you can save it and reuse it.

We can include a small description for each role above so that the user knows what's possible for each.

-----

In addition to this, I think it would also be nice to add the concept of a "Display Name" when adding a new member. That way it's easy to validate who is who and who has what permissions (vs looking at different wallet addresses)

I've mocked something up in #202 , but it's currently just commented out. It would look something like this (though the user-specific custom avatars are not available, yet)
<img width="481" alt="image" src="https://user-images.githubusercontent.com/80348806/214683392-76e53788-8372-4bf2-ae31-80f4162ddee1.png">

**Comment by auryn-macmillan** [comment]
Yeah, I think display names are a good idea. It kind of hints at #172, which I think we decided is out of scope for now. If we do add display names, we should be conscious of how this might interact/be used as part of #172 in future.

Otherwise, yeah I agree that "permissions" could be changed to "members" or "team", with each members role/permissions displayed alongside their address/display name. This begs the question of how to handle/display custom permission configurations.
